components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/ConfigurationLoader.java

```diff
@@ -161,6 +160,49 @@ public static void loadInternalApis(String apiFilePath) {
         }
     }
 
+    private static void setHandlers(OMElement apiElement, InternalAPI api) {
+
+        List<InternalAPIHandler> handlerList = new ArrayList<>();
+        OMElement handlersOM = apiElement.getFirstChildWithName(new QName("handlers"));
```

Comment:
handlers QName can be defined at the top (same as API_Q)

Reply:
Fixed